repo/config/profile.go

```diff
@@ -41,4 +41,11 @@ var ConfigProfiles = map[string]func(*Config) error{
 		c.Discovery.MDNS.Enabled = false
 		return nil
 	},
+	"badgerds": func(c *Config) error {
+		c.Datastore.Spec = map[string]interface{}{
+			"type": "badgerds",
+			"path": "badgerds",
+		}
+		return nil
+	},
 }
```

Comment:
I may be missing something, but won't this config bypass the measure datastore?  If so will this create a problem?

Reply:
IIRC there is global measure in fsrepo